libs/ml/examples/mnist/main.cpp

```diff
@@ -39,16 +40,16 @@ using SizeType   = fetch::math::SizeType;
 
 using GraphType      = typename fetch::ml::Graph<TensorType>;
 using OptimiserType  = typename fetch::ml::optimisers::AdamOptimiser<TensorType>;
-using DataLoaderType = typename fetch::ml::dataloaders::MNISTLoader<TensorType, TensorType>;
+using DataLoaderType = typename fetch::ml::dataloaders::TensorDataLoader<TensorType, TensorType>;
 
 int main(int ac, char **av)
 {
   DataType                      learning_rate{0.01f};
   SizeType                      subset_size{100};
   SizeType                      epochs{10};
-  SizeType                      batch_size{10};
+  SizeType                      batch_size{13};
```

Comment:
why?

Reply:
When I'm debugging a graph then using prime numbers is helpful because it makes it clear which parameters affect the size of a tensor! But it's debug code and I'll remove it.

Reply:
fancy!